internal/start/start.go

```diff
@@ -1143,6 +1143,10 @@ EOF
 		if err != nil {
 			return err
 		}
+
+		// Mount snippets directory for Studio to access
+		hostSnippetsPath := filepath.Join(workdir, utils.SnippetsDir)
+		binds = append(binds, fmt.Sprintf("%s:%s:rw", hostSnippetsPath, hostSnippetsPath))
```

Comment:
```suggestion
		binds = append(binds, fmt.Sprintf("%s:%s:rw", hostSnippetsPath, hostSnippetsPath))
		binds = utils.RemoveDuplicates(binds)
```

handle the rare case where function imports the snippets directory